src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Browser.cs

```diff
@@ -35,9 +35,7 @@ private static unsafe int GetFirstDayOfWeek(string localeName)
             int result = Interop.JsGlobalization.GetFirstDayOfWeek(localeName, out int exception, out object ex_result);
             if (exception != 0)
             {
-                // Failed, just use 0
                 Debug.Fail($"[CultureData.GetFirstDayOfWeek()] failed with {ex_result}");
-                return 0;
```

Comment:
this change seems to be masking the issue, rather than fixing it. Let's figure out what's the root cause.